vmm/src/vm.rs

```diff
@@ -614,11 +630,36 @@ impl Vm {
 
     #[cfg(target_arch = "aarch64")]
     fn configure_system(&mut self, _entry_addr: EntryPoint) -> Result<()> {
-        let _cmdline_cstring = self.get_cmdline()?;
-        // What to do on AArch64 is:
-        // - Setup GIC
-        // - Generate FDT
-        unimplemented!();
+        let cmdline_cstring = self.get_cmdline()?;
+        let vcpu_mpidrs = self.cpu_manager.lock().unwrap().get_mpidrs();
+        let guest_memory = self.memory_manager.lock().as_ref().unwrap().guest_memory();
+        let mem = guest_memory.memory();
+
+        let device_info = &self
+            .device_manager
+            .lock()
+            .unwrap()
+            .get_device_info()
+            .clone();
+
+        arch::configure_system(
+            &self.memory_manager.lock().as_ref().unwrap().fd,
```

Comment:
Echoing comment https://github.com/cloud-hypervisor/cloud-hypervisor/pull/1126#discussion_r438113995
The `fd` was added for this. Now I set `fd` field public in memory manager and BORROW it back from there.